bin/database.py

```diff
@@ -22,7 +22,7 @@
 from api.types import Origin
 from api.jobs import batch
 
-CURRENT_DATABASE_VERSION = 37 # An int that is bumped when a new schema change is made
+CURRENT_DATABASE_VERSION = 38 # An int that is bumped when a new schema change is made
```

Comment:
Just one question: Do we need to make this 39, or are the ops team just going to backrev this integer?

Reply:
In the PR description I mentioned you can use the `--from` flag to upgrade from a specific version. Rerunning 37 and 38* if either had been run before will not cause issue. Upgrades are intended to be written in a way that they can be rerun, usually in a way that they return an empty cursor and move on.

Reply:
Thanks Megan.

Reply:
I don't see that on this PR ticket, I'm guessing you meant the prior PR. SGM though.